src/main/java/zd/zero/waifu/motivator/plugin/assets/VisualMotivationAssetProvider.kt

```diff
@@ -17,6 +17,7 @@ enum class WaifuAssetCategory {
     DEPARTURE,
     ENCOURAGEMENT,
     TSUNDERE,
+    MOCKING,
```

Comment:
I added this, but did not know the best place to put this.

Should it be part of a normal test failure, or should a specific chain of events or condition happen in order for WENDI to mock you?

Reply:
> This is fine too, as it'll be a random event.

I just threw it in a chance to happen if WENDI is `Disappointed`, I think we can write another story to so that it can happen more often, less often, or not at all, kind of like a small part of #23 ?